MyPlayground.playground/Pages/Optional.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,72 @@
+//: [Previous](@previous)
+
+import Foundation
+
+print("Optional Declarre")
+var maybeANumber = "19375"
+var number:Int? = Int(maybeANumber)
+print(maybeANumber)
+
+//forced unwraping
+print("")
+var myString:String?
+
+myString = "Hello, Swift 4!"
+
+if myString != nil {
+   print( myString! )
+} else {
+   print("myString has nil value")
+}
+//auto unwrapping
+/*
+ var myString:String!
+ myString = "Hello, Swift 4!"
+ if myString != nil {
```

Comment:
remove commented code, if it is needed then write down proper comment above that